framework/web/HeadersAlreadySentException.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\web;
+
+use yii\base\Exception;
+
+/**
+ * HeadersAlreadySentException represents an exception caused by
+ * any headers that were already sent before web response was sent.
+ *
+ * @author Dmitry Dorogin <dmirogin@ya.ru>
+ * @since 2.0.14
+ */
+class HeadersAlreadySentException extends Exception
+{
+    /**
+     * Create instance of exception depending on debug mode.
+     *
+     * @param string $file
+     * @param int $line
+     * @return HeadersAlreadySentException
+     */
+    public static function make($file, $line)
```

Comment:
Why not using constructor?

Reply:
I dont want to change constructor params.

Reply:
Why?

Reply:
Just prefer this style. But in this case it's redundant.